server/web/templates/proposals/submissions-list.handlebars

```diff
@@ -134,7 +137,28 @@
                                                 {{/if}}                                                 
                                                 <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#reviewersList-{{_id}}">
                                                     See Reviewers
-                                                </button>                                                
+                                                </button>
```

Comment:
Why do we care if it has a feedback? if a proposal has uploaded revised proposals, it means it has final review status of revise and it means it absolutely has a feedback.